### ISSUE TITLE
Single consumer with multiple topics

### ISSUE THREAD

**Issue by jainanda-k** [karafka/karafka#555]
I have the following settings for the consumer in karafka.rb(1.2.12).

  ```
consumer_groups.draw do
  topic :test_topic do
      consumer TestConsumer
    end

    topic :test_topic1 do
      consumer TestConsumer
    end
end
```
Whenever I produce a message, the message always goes to the last topic. For, eg,when I produce a message for _test_topic_, the method for _test_topic1_ is called.

**Comment by chaadow** [comment]
You use the same consumer for both topics `TestConsumer`. If you want separate logic, consider adding a distinct Karafka Consumer class

**Comment by mensfeld** [comment]
Hey @jainanda-k sorry for the delay and @chaadow thank you for help. Dunno how I missed that :(

This is fixed in Karafka 1.3: https://mensfeld.pl/2019/09/karafka-framework-1-3-0-release-notes-ruby-kafka/#single-consumer-class-supports-more-than-one-topic
